packages/slate-react/src/plugins/after.js

```diff
@@ -305,8 +305,8 @@ function AfterPlugin() {
 
     // Get the selection point.
     const native = window.getSelection()
-    const { anchorNode, anchorOffset } = native
-    const point = findPoint(anchorNode, anchorOffset, value)
+    const { anchorNode } = native
+    const point = findPoint(anchorNode, 0, value)
```

Comment:
Why this change?

Reply:
Consider a leaf can be extended by a spell check:
```
<b>hel</b> yourself
```
will be 
```
<b>help</b> yourself
```
after spell check.  Because `anchorOffset` is measured after the spell check.  It will tell that anchorOffset is `4`.  However, during `onInput`, the text is not yet changed.  `offset =4` will tell the slate the spell corrected leaf is `yourself`, not `hel`